iocage/lib/Release.py

```diff
@@ -589,6 +591,7 @@ def update(
 
         jail = iocage.lib.Jail.JailGenerator(
             {
+                "name": self.name.replace(".", "-"),
```

Comment:
this commit needs a bit more elaboration…
what's the benefit of giving it a name, why are '.' bad?